src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
About point 1) maybe a unique sleep from the time the GATE job ends (obtained with getClock) and the time at which the Merge job starts would be cleaner. In fact I'm not even sure we need something here. when GATE jobs end they upload their files and are stopped. The merge starts when it has to and reads what's available. What was the motivation for a timer here?